src/sql/workbench/contrib/notebook/browser/notebook.css

```diff
@@ -7,10 +7,6 @@
 	padding-top: 16px;
 }
 
-.notebookEditor .taskbarSeparator {
```

Comment:
Can we keep the separator height as it was to match the original designs?

Reply:
I think all the toolbars in ADS should have the same style, personally I prefer to have some margin, for example:
![image](https://user-images.githubusercontent.com/13777222/124798018-c3f04280-df07-11eb-9a5e-b7a419506585.png)

@radentonev , what do you think?